requirements/prod-requirements.in

```diff
@@ -1,4 +1,4 @@
--r requirements.txt
+-r requirements.in
```

Comment:
I made this change because requirements/requirements.txt contains futures, which does not exist in Python 3.  This change has a beneficial side effect of adding ```# via``` comments to requirements/prod-requirements.txt.  It annoyed me in the past that those comments were absent, so I'm considering this an improvement.